pkg/aws/aws.go

```diff
@@ -38,28 +41,74 @@ const (
 )
 
 type awsCloud struct {
-	client  awsClient.Interface
-	infraID string
-	region  string
+	client               awsClient.Interface
+	infraID              string
+	region               string
+	nodeSGSuffix         string
+	controlPlaneSGSuffix string
 }
 
 // NewCloud creates a new api.Cloud instance which can prepare AWS for Submariner to be deployed on it.
 func NewCloud(client awsClient.Interface, infraID, region string) api.Cloud {
-	return &awsCloud{
+	awsCloud := &awsCloud{
 		client:  client,
 		infraID: infraID,
 		region:  region,
 	}
+
+	err := awsCloud.setSuffixes()
+	if err != nil {
+		return nil
+	}
+
+	return awsCloud
+}
+
+func (ac *awsCloud) setSuffixes() error {
+	vpcID, err := ac.getVpcID()
```

Comment:
Should we also log the error  (could be useful for debugging failures) ?

Reply:
not relevant now.